plugin/ochttp/trace_test.go

```diff
@@ -92,10 +92,14 @@ func TestTransport_RoundTrip_Race(t *testing.T) {
 		Base:        transport,
 	}
 	req, _ := http.NewRequest("GET", "http://foo.com", nil)
+
```

Comment:
Could we add a synchronizer to ensure that the racer goroutine terminates
before we exit the test?

```go
var wg sync.WaitGroup
wg.Add(1)

go func() {
   defer wg.Done()
   
   for i := 0; i < 1000; i++ {
     req.Header.Add("trace", "another one")
   }
}()
rt.RoundTrip(req)

_ = <-transport.ch
wg.Wait()
```

Reply:
We still cannot guarantee if the gorotutine started before RoundTrip returned. We are trying to create a race that is hard to produce here.